Solutions/PostgreSQL/Data Connectors/Connector_PostgreSQL.json

```diff
@@ -0,0 +1,168 @@
+        "instructionSteps": [
+                {
+                "title": "Install agent on Azure Windows Virtual Machine",
+                "description": "Select the machine to install the agent on and then click **Connect**.",
+                "instructions": [
+                    {
+                    "parameters": {
+                    "linkType": "InstallAgentOnVirtualMachine"
+                    },
+                    "type": "InstallAgent"
+                    }
+                ]
+            },
+                {
+                "title": "Install agent on a non-Azure Windows Machine",
+                "description": "Download the agent on the relevant machine and follow the instructions.",
+                "instructions": [
+                    {
+                    "parameters": {
+                    "linkType": "InstallAgentOnNonAzure"
+                    },
+                    "type": "InstallAgent"
+                    }
+                ]
+            }
+        ]
+            },
+                "type": "InstructionStepsGroup"
+                }
+            ]
+                },
+    {
+                    "title": "2. Configure PostgreSQL to write logs to files",
+                    "description":"1. Edit postgresql.conf file to write logs to files:\n\n>**log_destination** = 'stderr'\n\n>**logging_collector** = on\n\nSet the following parameters: **log_directory** and **log_filename**. Refer to the [PostgreSQL documentation for more details](https://www.postgresql.org/docs/current/runtime-config-logging.html)"
+                },
+    {
+            "title": "3. Configure the logs to be collected",
+            "description": "Configure the custom log directory to be collected" ,
+            "instructions": [
+                {
+                    "parameters": {
+                        "linkType": "OpenCustomLogsSettings"
+                    },
+                    "type": "InstallAgent"
+                }
+            ]
+        },
+    {
+        "title": "",
+        "description":"1. Select the link above to open your workspace advanced settings \n2. From the left pane, select **Settings**, select **Custom Logs** and click **+Add custom log**\n3. Click **Browse** to upload a sample of a PostgreSQL log file. Then, click **Next >**\n4. Select **Timestamp** as the record delimiter and click **Next >**\n5. Select **Windows** or **Linux** and enter the path to PostgreSQL logs based on your configuration(e.g. for some Linux distros the default path is /var/log/postgresql/) \n6. After entering the path, click the '+' symbol to apply, then click **Next >** \n7. Add **PostgreSQL** as the custom log Name (the '_CL' suffix will be added automatically) and click **Done**."
+    },
+    {
+        "title": "Validate connectivity",
+        "description":"It may take upwards of 20 minutes until your logs start to appear in Microsoft Sentinel."
+    }
+]
+}
```

Comment:
Please add the metadata for data connector

Reply:
metadata added